pkg/core/pipeline/condition/main.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/updatecli/updatecli/pkg/core/pipeline/scm"
 	"github.com/updatecli/updatecli/pkg/core/result"
 	"github.com/updatecli/updatecli/pkg/core/transformer"
-	"github.com/updatecli/updatecli/pkg/plugins/aws/ami"
+	"github.com/updatecli/updatecli/pkg/plugins/awsami"
```

Comment:
What is the motivation to rename from aws/ami ? when I decided on the name of that plugin, I was envisioning to have sub-packages for other AWS "services".

Reply:
The name `ami` misses context for someone not as ease with AWS. Hence the proposal to name it`awsami`.
Even if keeping the subdirectory `aws/` (which feels early optimization e.g. evilness ;) ), the name `ami` alone is not really helpful unless you really know what it is.


But I'm not strong on the renaming: if you disagree, then I'll remove it from the PR (I don't mind  since it is nipicking from me)

Reply:
> The name ami misses context for someone not as ease with AWS
Maybe we just need more documentation

> Even if keeping the subdirectory aws/ (which feels early optimization e.g. evilness ;) ), the name ami alone is not really helpful unless you really know what it is.
It was definitely early optimization 😆 , 

> But I'm not strong on the renaming: if you disagree, then I'll remove it from the PR (I don't mind since it is nitpicking from me)
Me neither, when I created this package, I hesitated about having `aws/ami` or `awsami`. I found it clearer the subpackage but indeed It's nitpicking. I let you make the final decision as you'll be the one merging this pull request :)

Reply:
I've merge with the new name: it means that, when the time of adding another AWS resource will come, I'll have to re-rename (you know where I live :D )